recipe/meta.yaml

```diff
@@ -73,21 +68,51 @@ test:
     - pytest-xdist         # [CI != "travis"]
     - hypothesis
     - setuptools <60.0.0
-    # some linux tests need a compiler
-    - {{ compiler('c') }}  # [linux]
+    - typing_extensions
+    # some linux tests need a C compiler;
+    # extra f2py tests need a fortran compiler
+    - {{ compiler('c') }}        # [linux]
+    - {{ compiler('fortran') }}  # [linux]
   commands:
     - f2py -h
     # numpy.test will show SIMD features of agent (in the past, there have been
     # failures that occured depending on presence/absence of e.g. AVX512);
     # for signature of numpy.test see the following (note default: label='fast'),
     # https://github.com/numpy/numpy/blob/maintenance/1.22.x/numpy/_pytesttester.py#L81-L82
-    {% set label = "'full'" if (CI != "travis") else "'fast'" %}
+    {% set label = "'fast'" if (CI == "travis") or (aarch64 and python_impl == "pypy") else "'full'" %}
```

Comment:
How much extra test time does this cost? I could do the "math" here, but it's not trivial to figure out on what platforms this now switches from "fast" to "full".

Reply:
It just inverted the condition for easier readability, and then added aarch+pypy because that was timing out with `'full'` also on azure.